docs/reference/security.md

```diff
@@ -111,70 +111,20 @@ Here's an example .vsls.json file:
 ```json
 {
     "$schema": "http://json.schemastore.org/vsls",
-    "gitignore":"none",
     "excludeFiles":[
         "*.p12",
         "*.cer",
         "token",
         ".gitignore"
-    ],
-    "hideFiles": [
-        "bin",
-        "obj"
-    ]
+    ] 
 }
 ```
 
 > [!NOTE]
 > You can also make the all files/folders you share **read-only** when you start a collaboration session. See [below](#read-only-mode) for details.
 
-Let's walk through how these properties change what guests can do.
-
-### Properties
-
 The **excludeFiles** property allows you to specify a list of glob file patterns (very much like those found .gitignore files) that prevents Live Share from opening certain files or folders for guests. Be aware that this is inclusive of scenarios like a guest _following or jumping to your edit location, stepping into a file during collaborative debugging, any code navigation features like go to definition, and more._ It is intended for files you never want to share under any circumstances like those containing secrets, certificates, or passwords. For example, since they control security, .vsls.json files are always excluded.
```

Comment:
- It might be important to call out that exclude files not only works at the root folder of the directory since filtering no longer works

Reply:
I added a line - will highlight and comment so you can read it.